q2_phylogeny/tests/test_raxml.py

```diff
@@ -25,16 +26,22 @@ class RaxmlTests(TestPluginBase):
 
     package = 'q2_phylogeny.tests'
 
+    def setUp(self):
+        super().setUp()
+        src = Path(__file__).parent / "data"
```

Comment:
The base class you're overriding uses [`pkg_resources`](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resource-extraction) for this, I would prefer if we kept it that way (I believe we did this to ensure that we were looking at the packaged files, rather than the source files).

```suggestion
        pkg_resources.resource_filename(self.package, 'data/')
```

Although, I'm not certain that this API will allow you to look up a directory like I am suggesting above. If not, we should circle back on this.